### PR TITLE
Feat: Inputs

### PR DESCRIPTION
**Pull request by psychoactiv**
### Inputs
Added various types of Inputs

13 02 2022_03 10 06_REC
13 02 2022_03 10 27_REC
13 02 2022_03 10 56_REC

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
good to see that your component has javascript also

File: components/Inputs/inputs.css [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+.chip-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 2rem 0;
+  border-radius: 0.5rem;
+  margin-top: 5rem;
+  width: 50rem;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: aliceblue;
```
Comment (tusharkandpal) [inline components/Inputs/inputs.css:17]: color variable is preferred

File: components/Inputs/inputs.css [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+.ring-cta {
+  margin-left: auto;
+  border-top: 0.9px solid var(--black);
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 0.8rem;
+  padding-top: 0.5rem;
+}
+
+.chosenRingtone {
```
Comment (tusharkandpal) [inline components/Inputs/inputs.css]: your naming conventions should be consistent.

File: components/Inputs/inputs.html [hunk 3]
```diff
@@ -0,0 +1,1015 @@
...
+      crossorigin="anonymous"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/atom-one-dark-reasonable.min.css"
+      integrity="sha512-RwXJS3k4Z0IK6TGoL3pgQlA9g2THFhKL7z9TYWdAI8u6xK0AUuMWieJuWgTRayywC9A94ifUj1RzjDa1NIlUIg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Document</title>
```
Comment (tusharkandpal) [inline components/Inputs/inputs.html]: change the title accordingly

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
please work on the review comments below

File: components/Inputs/inputs.css (hunk 2 above)
Comment (psychoactiv) [reply components/Inputs/inputs.css]: Will do that

File: components/Inputs/inputs.css (hunk 1 above)
Comment (psychoactiv) [reply components/Inputs/inputs.css:17]: I'll change those.

File: components/Inputs/inputs.html (hunk 3 above)
Comment (psychoactiv) [reply components/Inputs/inputs.html]: Sure

**Pull request merged**